app/assets/javascripts/discourse/app/widgets/decorator-helper.js

```diff
@@ -42,6 +42,10 @@ class DecoratorHelper {
     return this.widget.attach(name, attrs, state);
   }
 
+  get model() {
```

Comment:
How about giving this a more descriptive name like `.post`? (for both new and old decorator helpers)

Reply:
Unfortunately, the model is not necessarily a post.

Currently, we only use it with a Post that comes from the PostCookedHtml component, but the helper is from the DecoratedHtml component which is neutral and can be used as a base to another component which could use another type of model: a chat message for example